Invalid flag: FATAL:/opt/local/libexec/darwin-xtools/bin/./ppc/as: I don't understand '-' flag!

--- auxil/spicy/3rdparty/CMakeLists.txt	2025-08-19 02:02:17.000000000 +0800
+++ auxil/spicy/3rdparty/CMakeLists.txt	2025-08-20 12:19:40.000000000 +0800
@@ -34,7 +34,6 @@
 # fiber library contains at least one) need an executable stack; explictly
 # disable that since we do not need executable stacks and some package linters
 # call this out as needlessly insecure.
-set(CMAKE_ASM_FLAGS ${CMAKE_ASM_FLAGS} -Wa,--noexecstack)
 
 set(FIBER_SHARED OFF)
 set(FIBER_OBJECT ON)
